app/jenkins-for-jira-ui/src/components/ConnectionPanel/ConnectionPanel.tsx

```diff
@@ -4,21 +4,44 @@ import { ConnectionPanelMain } from './ConnectionPanelMain';
 import { ConnectionPanelTop } from './ConnectionPanelTop';
 import { ConnectedState } from '../StatusLabel/StatusLabel';
 import { connectionPanelContainer } from './ConnectionPanel.styles';
+import { JenkinsServer } from '../../../../src/common/types';
+import { getAllJenkinsServers } from '../../api/getAllJenkinsServers';
+
+// TODO - add DUPLICATE state once I'm pulling in new data from backend
```

Comment:
Added this and other todos to a todo list that I'm tackling as I work through this.